cypress/integration/page.spec.js

```diff
@@ -291,7 +320,7 @@ describe('', () => {
         button().should('have.text', 'Choose a user');
       });
 
-      it('should not show users list by default', () => {
+      it.skip('should not show users list by default', () => {
```

Comment:
This test is currently skipped. Ensure it is intentionally skipped or enable it if relevant.